### PR TITLE
Update ci setting

### PR DESCRIPTION
**Pull request by kan-bayashi**
- Changed conditions not to perform duplicated CI when making branch in the original repo
- Removed duplicated CI in circle CI

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,8 +9,9 @@ workflows:
     jobs:
       - test-ubuntu18
```
Comment (ShigekiKarita) [inline .circleci/config.yml:10]: As ubuntu18 is also tested in Github Actions, how about disabling this too?

**Review by ShigekiKarita — APPROVED** [review]

**Comment by kamo-naoyuki** [comment]
Could you update mergify.yml?

https://github.com/espnet/espnet/blob/master/.mergify.yml#L14-L15

File: .circleci/config.yml (hunk 1 above)
Comment (kan-bayashi) [reply .circleci/config.yml:10]: I think this is for use_conda=true so I keep it as is.

**Review by kamo-naoyuki — APPROVED** [review]

**Comment by kamo-naoyuki** [comment]
auto-merge doesn't work if mergify.yml is updated in the PR.

**Pull request merged**